uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/apis/entitlement-api_test.js

```diff
@@ -0,0 +1,88 @@
+define([
+  'mocks/data/entitlement-data',
+  'leave-absences/shared/apis/entitlement-api',
+], function (mockData) {
+  'use strict';
+
+  describe('EntitlementAPI', function () {
+    var EntitlementAPI, $httpBackend;
+
+    beforeEach(module('leave-absences.apis'));
+
```

Comment:
here the newline is separating methods from variable declaration and another method so will keep them